### PR TITLE
Various grammar and format tweaks

### PR DESCRIPTION
**Pull request by cherylhughey**
- [X] This PR handles the issue and requires no additional PRs.
- [x] You have validated the need for this change.

**What did this PR accomplish?**

- Various grammar and format tweaks

### CODE REVIEW

File: Testing_for_APIs.md [hunk 1]
```diff
@@ -60,10 +60,10 @@ Web authentication types are based on:
 
 ### Generic Testing Method
 
-Step 1: List endpoint and make different request method: Login with user and then using a spider tool to list the endpoints of this role.
-To examine the endpoints, need to make different request methods and then observe how the API behaves.
+Step 1: List endpoint and make different request method: Login with user profile and use a spider tool to list the endpoints of this role.
+To examine the endpoints, you will need to make different request methods and observe how the API behaves.
 
-Step 2: Exploit bugs- as know how to list endpoints and examine endpoints with HTTP methods at step 1, we will find some way to exploit bugs as some testing strategies below:
+Step 2: Exploit bugs - As know how to list endpoints and examine endpoints with HTTP methods at step 1, we will find some way to exploit bug. Some testing strategies are below:
```
Comment (kingthorin) [inline Testing_for_APIs.md:66]: I'd suggest not putting too much time into this one. It obviously needs a bunch of work.

File: checklist/Testing_Checklist.md [hunk 2]
```diff
@@ -37,7 +37,7 @@ Note: The `Status` column can be set for values similar to "Pass", "Fail", "N/A"
 | WSTG-IDNT-04      | Testing for Account Enumeration and Guessable User Account                 |        |       |
 | WSTG-IDNT-05      | Testing for Weak or Unenforced Username Policy                             |        |       |
 | **WSTG-ATHN**     | **Authentication Testing**                                                 |        |       |
-| WSTG-ATHN-01      | Testing for Credentials Transported over an Encrypted Channel              |        |       |
+| WSTG-ATHN-01      | Testing for Credentials Transported Over an Encrypted Channel              |        |       |
```
Comment (kingthorin) [inline checklist/Testing_Checklist.md:40]: This was already correct for the Chicago style of titles that the project uses.
https://capitalizemytitle.com/#Chicago

File: document/4-Web_Application_Security_Testing/02-Configuration_and_Deployment_Management_Testing/12-Test_for_Content_Security_Policy.md [hunk 3]
```diff
@@ -16,7 +16,7 @@ Content Security Policy supports directives which allow granular control to the
 
 ## How to Test
 
-To test for misconfigurations in CSPs look for insecure configurations by examining `Content-Security-Policy` HTTP response header or CSP `meta` element in a proxy tool:
+To test for misconfigurations in CSPs, look for insecure configurations by examining `Content-Security-Policy` HTTP response header or CSP `meta` element in a proxy tool:
```
Comment (kingthorin) [inline document/4-Web_Application_Security_Testing/02-Configuration_and_Deployment_Management_Testing/12-Test_for_Content_Security_Policy.md]:
```suggestion
To test for misconfigurations in CSPs, look for insecure configurations by examining the `Content-Security-Policy` HTTP response header or CSP `meta` element in a proxy tool:
```

File: template/999-Foo_Testing/1-Testing_for_a_Cat_in_a_Box.md [hunk 4]
```diff
@@ -57,7 +57,7 @@ If no cat reveals itself, it is unlikely there is a cat in the box. However, thi
 
 ### Use a Box Camera
 
-Set up a wireless camera inside the box to continuously monitor for cats inside. This test requires the following steps.
+Set up a wireless camera inside the box to continuously monitor for cats inside. This test requires the following steps:
```
Comment (kingthorin) [inline template/999-Foo_Testing/1-Testing_for_a_Cat_in_a_Box.md:60]: This was good as-is.

**Comment by cherylhughey** [comment]
Proof reading changes.

File: Testing_for_APIs.md (hunk 1 above)
Comment (kingthorin) [reply Testing_for_APIs.md:66]: That's likely why it's sitting in the root and not yet placed.

**Comment by kingthorin** [comment]
Please also add yourself to: https://github.com/OWASP/wstg/tree/master/document/1-Frontispiece#reviewers-or-editors

**Comment by kingthorin** [comment]
Did you close this on purpose?

**Comment by cherylhughey** [comment]
Sorry, I thought I was supposed to close it when done.

**Comment by kingthorin** [comment]
No the items from the review need to be addressed then we’ll merge the changes.

**Comment by kingthorin** [comment]
@cherylhughey so if you could just push additional changes addressing the review comments we will get this merged.

**Comment by cherylhughey** [comment]
I took a look and have no further changes to push. Thank you.

________________________________

**Review by kingthorin — APPROVED** [review]

**Review by ThunderSon — APPROVED** [review]

**Comment by kingthorin** [comment]
Thanks

**Pull request merged**